appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Verifies the given attestation passes the given rego policy
+# usage:
+#   verify-attestation-with-policy.sh <attestation file> <output file> <passed file>
+# where:
+#   <in-toto file> file containing the in-toto attestation to be verified
```

Comment:
Lines 5 and 7 should be using the same nomenclature. Either `attestation file` in both, or `in-toto file` in both.

Reply:
Dang it. Have to fix it later now..

Reply:
Fixed in https://github.com/redhat-appstudio/build-definitions/pull/62